### DISCUSSION TITLE
Remove stand-alone mode

### DISCUSSION THREAD

**Discussion by kangz12345** [snu-quiqcl/qiwis#43] [Ideas]
As mentioned in https://github.com/snu-quiqcl/swift/issues/40#issuecomment-1456238620, the stand-alone execution mode seems not necessary.
How about removing that part of `numgen` and not implementing such part in the future applications as well?

**Comment by BECATRUE** [comment]
I agree it.

The stand-alone execution mode was implemented just for testing if `numgen` works correctly.
Now, it is not necessary.
